core/src/main/java/greencity/config/SecurityConfig.java

```diff
@@ -337,7 +337,8 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
                     "/habit/assign/delete/{habitAssignId}",
                     FRIENDS + "/{friendId}",
                     FRIENDS + "/{friendId}/cancelRequest",
-                    FRIENDS + "/")
+                    FRIENDS + "/",
+                    "/habit/delete/{customHabitId}")
```

Comment:
please add it after `"/habit/assign/delete/{habitAssignId}",` to have grouping by endpoints